test/addons-napi/test_conversions/test.js

```diff
@@ -116,7 +116,8 @@ assert.deepStrictEqual(new Boolean(false), test.toObject(false));
 assert.deepStrictEqual(new Boolean(true), test.toObject(true));
 assert.deepStrictEqual(new String(''), test.toObject(''));
 assert.deepStrictEqual(new Number(0), test.toObject(0));
-assert.deepStrictEqual(new Number(Number.NaN), test.toObject(Number.NaN));
+// TODO: Add test back in as soon as NaN is properly compared
+// assert.deepStrictEqual(new Number(Number.NaN), test.toObject(Number.NaN));
```

Comment:
Can we remove the todo and add the test back?

Reply:
Not before #15036 landed. Otherwise the test would fail.

Reply:
#15036

Reply:
Killswitch

Reply:
Don't Forget

Reply:
what's the depedency of this change to https://github.com/nodejs/node/pull/15036?

Reply:
Currently `assert.deepStrictEqual(NaN, NaN)` will fail. This test only worked because `NaN` was never compared as it was not unboxed before. #15036 will change that to not fail anymore in case `NaN` is compared to itself.

Reply:
Have you pushed you changes to this PR? Looks like these lines are still commented?

Reply:
Thanks a lot for reminding me! It is now back in place.